File: greedy.c
#include <cs50.h>
#include<stdio.h>

int main(void)
{
    float change;
    int cents,count=0;
    printf("Change : ");
    change = get_float();
    cents = change*100;
    while(cents)
    {
        if(cents>25)
        {
            cents = cents - 25;
            count++;
        }
        else if(cents>10)
        {
            cents = cents - 10;
            count++;
        }
        else if(cents>5)
        {
            cents = cents - 5;
            count++;
        }
        else
        {
            cents = cents - 1;
            count++;
        }
    }
    printf("%d\n",count);
}

File: greedy.py
import cs50

def main():
    while True:
        print("O hai! How much change is owed?")
        change = cs50.get_float()
        if(change>=0):
            break
    cents = change * 100
    count = 0
    while cents:
        if(cents>=25):
            cents = cents - 25
            count = count + 1
        elif(cents>=10):
            cents = cents - 10
            count = count + 1
        elif(cents>=5):
            cents = cents - 5
            count = count + 1
        else:
            cents = cents - 1
            count = count + 1
    print("{}".format(count))
        
    
if __name__ == "__main__":
    main()
